packages/react/src/components/UIShell/HeaderPanel.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState, useRef } from 'react';
+import { settings } from 'carbon-components';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { AriaLabelPropType } from '../../prop-types/AriaPropTypes';
+
+const { prefix } = settings;
+
+const HeaderPanel = React.forwardRef(function HeaderPanel(props, ref) {
+  const {
+    'aria-label': ariaLabel,
+    'aria-labelledby': ariaLabelledBy,
```

Comment:
are these aliases still needed if we can just use the aria attributes directly?

Reply:
I was going by the code already there, since the existing components already have them.